### DISCUSSION TITLE
2024/01/15/factorio-server/

### DISCUSSION THREAD

**Discussion by giscus** [DancingSnow0517/DancingSnow0517.github.io#5] [Announcements]
# 2024/01/15/factorio-server/

此教程基于linux服务器，使用windows类似  下载服务器首先，下载异星工厂的headless版本，可以在 这里 下载，上传到服务器中的一个地方 或者直接使用命令下载到服务器 wget https://factorio.com/get-download/stable/headless/linux64  配置服务器下载的服务端的文件名字为 factorio_headless_x64_X.X.X

https://blog.dancingsnow.xyz/2024/01/15/factorio-server/

**Comment by ZhuRuoLing** [comment]
喵~
